bridge/client/app/_components/ktb-user/ktb-user.component.ts

```diff
@@ -0,0 +1,15 @@
+import {Component, Input} from '@angular/core';
+
+@Component({
+  selector: 'ktb-user',
+  templateUrl: './ktb-user.component.html',
+  styleUrls: ['./ktb-user.component.scss']
+})
+export class KtbUserComponent {
+
+  @Input() user;
+
+  logout(): void {
+    window.location.href = '/logout';
```

Comment:
That's not what I would expect in an Angular App. Could you explain why we are using `window.location.href` here instead of using routes/states to perform a logout?

Reply:
Because the `/logout` route is not handled by the Angular app, but only by the node app and removes the session in the backend. We could add a route in the Angular app and make an API call there, but I don't know if we really need that?

Reply:
Okay, then let's add this as a comment:
```
// redirect user to the logout page, which is handled by the express app
```
One further question:
What if `bridge` is hosted behind a proxy, e.g., https://myserver.com/my-namespace/keptn/bridge/? In this case navigating to `/logout` would mean that we navigate to https://myserver.com/logout, instead of https://myserver.com/my-namespace/keptn/bridge/logout.

Reply:
@christian-kreuzberger-dtx Thanks for pointing this out.

Latest commit address this issue in both angular and express layer

For angular, since we have defined the base-href , I now use Location service [1]

For express, I have a conditional navigation. PREFIX_PATH was introduced to bridge and if it is present, root navigation is altered to PREFIX_PATH/bridge. 

please have a look on the changes. 

[1] - https://angular.io/api/common/Location

Reply:
Nice, thanks